docs/content/en/blog/releases/v0.9.11.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Release v0.9.11"
+linkTitle: "Release v0.9.11"
+date: 2021-04-12
+description: >
+  Release v0.9.11
+---
+
+## Changelog since v0.9.10
+
+### Notable Changes
+* Disable pipectl datastore command ([#1843](https://github.com/pipe-cd/pipe/pull/1843))
+* Disable MongoDB as ControlPlane's datastore ([#1835](https://github.com/pipe-cd/pipe/pull/1835))
+
```

Comment:
Could you please add "Piped now works with namespace scope resources as well as cluster scope."?

Reply:
Where is the PR for that?

Reply:
Here: https://github.com/pipe-cd/pipe/pull/1830

Reply:
Already exists at L24: `Update piped to not create cluster-level resources ([#1830](https://github.com/pipe-cd/pipe/pull/1830))`

Reply:
OK. From the next times, you can fill that description into the PR's body at
`Does this PR introduce a user-facing change?:`

Reply:
Right. I have totally forgotten. I'm going to do so.

Reply:
👍  I have just fixed the changelog.